en/theme/material/assets/css/theme.css

```
@@ -376,397 +436,489 @@
 .md-footer__content,
 .md-copyright__content {
-    max-width: none !important;
-    padding: .5rem 1.7rem;
-    display: grid !important;
-    grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
-    grid-gap: var(--md-footer-section-horizontal-spacing);
+  max-width: none !important;
+  padding: 0.5rem 1.7rem;
+  display: grid !important;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  grid-gap: var(--md-footer-section-horizontal-spacing);
```

Stylelint (17.14.0)

[error] Expected "grid-gap" to be "gap" (property-no-deprecated)

(property-no-deprecated)